manifests/bootable-rpm-ostree.yaml

```
@@ -13,16 +13,14 @@ packages:
  - linux-firmware
  # rpm-ostree
  - rpm-ostree nss-altfiles
-
+ - fwupd
```

Add the comment and also the newline back:

```suggestion
 # firmware updates
 - fwupd
 
```